### PR TITLE
Containerize

### PR DESCRIPTION
**Pull request by rosspenna**
### What
  Added `Dockerfile` and scripts to build/run docker image/container locally.

### Why
 https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/issues/227

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [x] Tapping on a marker on the map displays information about the marker in a popup.
- [ ] Tapping the "Show list of locations" button replaces the map view with a list view.
- [x] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [x] Tapping one of the ✅'s in the legend filters items on the map and in the list.
- [x] Changing the language to spanish changes things on the page.
- [x] Clicking the Help/Info button opens and closes a menu with information.
- [x] Clicking the X Close button on the Help/Info screen closes the modal.

### Check the app in the following web browsers:
- [x] Chrome
- [x] Firefox
- [ ] Edge
- [ ] Safari

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,11 +56,35 @@ This started as a very lightweight, single page html file, and we've tried very
...
+3. build the image
+    ```
+    docker-build.sh
+    ```
+    * image base: https://hub.docker.com/_/node/
+    * will fail if there is not a tagged image for the current node version specified in [./node-version](./node-version)
+    * optional argument specifies "IMAGE_VERSION" (defaults to "latest" if not supplied)
+4. run the image
+    ```
+    docker-build.sh
```
Comment (adyates) [inline README.md:73]: I think you meant `docker-run.sh` here =)

**Review by adyates — COMMENTED** [review]
Only thing I might add is a quick note in the README about how one can see the stdout of the container after you run it in a detached mode (e.g. `docker-compose logs -f web` if I'm reading this correctly), assuming the app does that.

Otherwise 👍

**Comment by amaxama** [comment]
@rosspenna @adyates Any movement on this PR?

**Review by adyates — APPROVED** [review]
Apologies!  LGTM =)

**Pull request closed without merging**